sections/clause_07a_sensing_core_entities.adoc

```diff
@@ -37,10 +34,11 @@ image::images/GRP0001.png[Sensing Core, align="center"]
 In this section, we define each entity depicted in <<img-sta-core>> and its relationships with other entities.
 Additionally, we also provide examples to model the entities in different contexts.
 
-[[thing]]
-==== Thing
+==== Requirement Class: Thing
```

Comment:
The [[thing]] anchor allows referencing this section. Not sure if we ever need to though, can always re-add the anchors.

Reply:
merge issue, will fix. Thanks for pointing!